quality.py

```diff
@@ -31,27 +34,37 @@
 _DEPENDS = ['state']
 
 
-class Proof(ModelSQL, ModelView):
-    'Quality Proof'
-    __name__ = 'quality.proof'
-
-    name = fields.Char('Name', required=True, translate=True,
-        select=True)
+class ProofMixin(Model):
```

Comment:
Better remove the required flag from company field in quality.proof.

Creating the template + wizard is too complex.